master/internal/db/postgres_test_utils.go

```diff
@@ -185,6 +185,124 @@ func RequireMockJob(t *testing.T, db *PgDB, userID *model.UserID) model.JobID {
 	return jID
 }
 
+// RequireMockCommandID creates a mock command and returns a command ID.
+func RequireMockCommandID(t *testing.T, db *PgDB, userID model.UserID) model.TaskID {
+	task := RequireMockTask(t, db, &userID)
+	alloc := RequireMockAllocation(t, db, task.TaskID)
+
+	mockCommand := struct {
+		bun.BaseModel `bun:"table:command_state"`
+
+		TaskID             model.TaskID
+		AllocationID       model.AllocationID
+		GenericCommandSpec map[string]any
+	}{
```

Comment:
thought: it would be nice if we could define this as `model.CommandSnapshot{...}` but I think we can't because of an import cycle.